apps/studio/components/interfaces/Database/Wrappers/WrappersDisabledState.tsx

```diff
@@ -24,34 +33,46 @@ const WrappersDisabledState = () => {
     'extensions'
   )
 
+  const { mutateAsync: enableExtension } = useDatabaseExtensionEnableMutation()
+
   const onEnableWrappers = async () => {
     if (wrappersExtension === undefined || vaultExtension === undefined) return
+    if (project === undefined) return console.error('Project is required')
+    if (project.connectionString === undefined)
+      return console.error('Connection string is required')
+
     setIsEnabling(true)
 
     const requiredExtensions = await Promise.all([
-      await meta.extensions.create({
+      await enableExtension({
+        projectRef: project.ref,
+        connectionString: project.connectionString,
         schema: wrappersExtension.schema ?? 'extensions',
         name: wrappersExtension.name,
         version: wrappersExtension.default_version,
         cascade: true,
       }),
-      await meta.extensions.create({
+      await enableExtension({
+        projectRef: project.ref,
+        connectionString: project.connectionString,
         schema: vaultExtension.schema ?? 'vault',
         name: vaultExtension.name,
         version: vaultExtension.default_version,
         cascade: true,
       }),
     ])
     const errors = requiredExtensions.filter(
-      (res) => res.error && !res.error.message.includes('already exists')
+      // @ts-ignore
```

Comment:
I think these need the `@ts-ignore`s because `enableExtension()` won't return the error but will throw it, so you might have to add a try/catch here instead of accessing it via `.error` 👍